### DISCUSSION TITLE
Control 2nd circuit

### DISCUSSION THREAD

**Discussion by TheNablaOperator** [basti242/homeassistant_lg_therma_v_modbus#11] [Q&A]
Hello.
Is it possible to control the 2nd circuit (mix pump & mix valve) via modbus, when AWHP pump is off?
The integrated LG controller only let the 2nd circuit flow when, AWHP pump is on...

Thanks.

**Comment by basti242** [comment]
I have no information about this.Why would you do this?

**Reply by TheNablaOperator** [reply]
Using a buffer tank with 2nd heat source... 
So if it's possible to control the 2nd circuit without having AWHP pump running, I wouldn't need extra euipment to control the valve and the 2nd circuit pump...
When the AWHP pump is running, it would empty the buffer tank by heating the outside (compressor)...
